profiler/cppcheck-suppressions.txt

```diff
@@ -0,0 +1,10 @@
+*:*/dotnet-runtime-coreclr/*
```

Comment:
Was this file in the original repo? I didn't see it anywhere

Reply:
yes, it was in the `.github/workflows`. I moved it in the profiler folder since it's related to the profiler only.